frontend/src/components/molecules/tasklist/TaskList.tsx

```diff
@@ -70,7 +73,7 @@ const TaskList = ({ isMobile }: { isMobile: boolean }) => {
     return parsedUrl.pathname + parsedUrl.search;
```

Comment:
If you go this way, I think you can (and should) also try to remove the workaround here as well; before you'd get double base URL's (so you removed them).

Now, you are making a query for a relative URL. Might as well query the full URL and have a bit less code.